libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -249,6 +249,21 @@
 #define HAL_BARO_MS5611_I2C_BUS 1
 #define HAL_BARO_MS5611_I2C_ADDR 0x77
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_HMC5843_MPU6000
+#define HAL_OPTFLOW_ONBOARD_VDEV_PATH "/dev/video0"
+#define HAL_OPTFLOW_ONBOARD_SENSOR_WIDTH 320
+#define HAL_OPTFLOW_ONBOARD_SENSOR_HEIGHT 240
+#define HAL_OPTFLOW_ONBOARD_OUTPUT_WIDTH 64
+#define HAL_OPTFLOW_ONBOARD_OUTPUT_HEIGHT 64
+#define HAL_OPTFLOW_ONBOARD_CROP_WIDTH 240
+#define HAL_OPTFLOW_ONBOARD_CROP_HEIGHT 240
+#define HAL_OPTFLOW_ONBOARD_NBUFS 8;
+#define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
+#define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
+#define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_PARAM_DEFAULTS_PATH "/etc/arducopter/minlure.parm"
+/* focal length 3.6 um, 2x binning in each direction
+ * 240x240 crop rescaled to 64x64 */
+#define HAL_FLOW_PX4_FOCAL_LENGTH_MILLIPX (2.5 / (3.6 * 2.0 * 240 / 64))
```

Comment:
Ok, so you have a focal length of 2.1mm and a pixel size of 3µm
For binning, it is a kind of scaling you have with camera sensors. You probably have some scaling if the camera is keeping is field of view while reducing the size of the image, but it might be hard to figure out the exact ratio. Maybe simply image_size/max_image_size but there might also be some cropping associated with it. You could determine it by taking pictures of an object with max resolution and the one you use and compare the resulting images (without moving the camera and the object of course).